libs/ui/src/components/MediaItem/MediaItem.vue

```diff
@@ -90,13 +101,19 @@ const components = {
 }
 
 const resolveComponent = computed(() => {
-  return components[resolveMedia(mimeType.value) + SUFFIX]
+  let mediaType = resolveMedia(mimeType.value)
+
+  if (mediaType === MediaType.IFRAME && !props.isDetail) {
+    isInteractive.value = true
+    mediaType = MediaType.IMAGE
+  }
+  return components[mediaType + SUFFIX]
 })
```

Comment:
![821wmj](https://github.com/kodadot/nft-gallery/assets/22471030/e9f6ad03-0656-4b3d-978d-fda4607b4532)

Reply:
Except for the NFT details page, all other iframe would be displayed as images

Reply:
Amazing!